lib/App.js

```diff
@@ -3101,19 +3101,17 @@ You can create an API Key on https://platform.openai.com/api-keys.`);
             if (file === 'locales') return;
             if (file === 'node_modules') return;
             if (file === '.homeybuild') return;
-            if (file === '.homeycompose') {
-              jsonFiles.add(path.normalize(`${filePath}/app.json`));
```

Comment:
Why is this removed? I think `.homeycompose/app.json` won't be translated now.

Reply:
This case was necessary as previously all `app.json` files were excluded, independently on which directory they were placed in. By removing [old line 3113](https://github.com/athombv/node-homey/pull/423/files#diff-b4a67a603557114ecf970e2cb40e5b018f858d8195591bbe969d3a91baa6c9f9L3113) it is now included by the normal file path.